### DISCUSSION TITLE
Can't get bitcoin core started

### DISCUSSION THREAD

**Discussion by mvezzuto** [cryptoadvance/specter-desktop#1818] [Help] [answered]
Hi everyone. I'm new to the thread...sent here by Moritz after I couldn't get on telegram because I don't have a smartphone (I'm SOMEWHAT off the grid). Anyway, I'm writing because I'm having trouble getting my Specter desktop app working after not having a problem with it for the 5 months or so I've been using it. First, I was getting an endless Pacman screen. I wound up deleting the 1.8.1 version (via App Zapper, which means a few other files went with it. I still have my code words and wallet backup, though) and updating the app to the most recent version. This didn't help, so on Moritz's advice, I went back to 1.8.1 (downloading the app saved on my backup disc from last June) and after 20 minutes of Pacman, it opened (I should mention, before Thursday July 21 I never had this problem with the Pacman app not disappearing. The app would usually open in less than 2 minutes). NOW I can't get the Bitcoin core to start. I tried starting the one I had, and when that didn't work, I tried configuring a new node, but got error messages from that (see photo below). The messages say "Encountered an error:Node specter_bitcoin_3_2 does not exist!). Any advice? I should mention that I have some experience in engineering (in college. I never worked in the field), so I'm not tech-phobic, but haven't kept up with developments in computers over the last 10 years, so some of the current jargon may go over my head. I can generally figure things out if pointed in the right direction. Anyway, thanks for reading, and thanks in advance for any help.
<img width="523" alt="Screen Shot 2022-07-22 at 21 17 58" src="https://user-images.githubusercontent.com/109803907/180442446-29c4ce92-dcf8-4e64-9ed6-fe9997d540d2.png">
p

**Comment by mvezzuto** [comment, marked as answer]
Also got this message when trying to configure the node via the wallet:
<img width="885" alt="Screen Shot 2022-07-22 at 23 42 35" src="https://user-images.githubusercontent.com/109803907/180464194-62e18f57-2857-4d92-8f7a-48fd20363c3d.png">

**Reply by mvezzuto** [reply]
Actually, never mind. Today, Apple sent a system update (Monterey 12.5), and the Specter app and Bitcoin Core are miraculously back to normal. Go figure! Sorry to take up your time!

**Reply by JS-Styles** [reply]
Interesting.  I recently started having issues with receiving Specter release updates through Umbrel node and wonder if it has to do with recent upgrade to Ventura OS.
